boards/silabs/dev_kits/xg24_ek2703a/Kconfig.defconfig

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2021, Sateesh Kotapati
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_XG24_EK2703A
+
+config CMU_HFXO_FREQ
+	default 39000000
+
+config CMU_LFXO_FREQ
+	default 32768
+
+if SOC_GECKO_USE_RAIL
+
+config FPU
+	default y
+
+endif # SOC_GECKO_USE_RAIL
+
+if BT
+
+config FPU
+	default y
+
+config COMMON_LIBC_MALLOC_ARENA_SIZE
+	default 8192
+
+config MAIN_STACK_SIZE
+	default 2304
```

Comment:
I'm curious, where does this value come from? Isn't the main thread's stack requirement completely dependent on the application, i.e. why do we need to set some special default on the board level?

Reply:
Agreed, why are these stack sizes here?

Reply:
This was a copy of the DK2601B config, I don't really understand why it has to be defined at the board level either, will remove it.